src/BackTranslationHelper/BackTranslationHelperCtrl.cs

```diff
@@ -829,8 +829,16 @@ private void TargetTextToolStripMenuItem_Click(object sender, EventArgs e)
                     textBoxPossibleTargetTranslation2.Font =
                     textBoxPossibleTargetTranslation3.Font =
                     textBoxTargetBackTranslation.Font = fontDialog.Font;
-                mapProjectNameToTargetFontOverride.Add(projectName, fontOverride);
-                Properties.Settings.Default.MapProjectNameToTargetFontOverride = SettingFromDictionary(mapProjectNameToTargetFontOverride);
+
+				if (mapProjectNameToTargetFontOverride.ContainsKey(projectName))
+				{
+					mapProjectNameToTargetFontOverride[projectName] = fontOverride;
```

Comment:
`dict[key] = value` should work whether or not the key exists. (no need for a the surrounding if statement)

Reply:
thanks. changed them all